### PR TITLE
Update Release-Process-Walkthrough.md

### PR DESCRIPTION
**Pull request by tomberek**
@jonringer @nrdxp

### CODE REVIEW

File: src/Release-Process-Walkthrough.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ git push upstream 20.03-pre
 ```
 
 11.  Contact the infrastructure team to create the necessary Hydra
-    Jobsets.
+    Jobsets. (nixos-21.11, nixos-21.11-small, nixos-21.11-aarch64)
```
Comment (cole-h) [inline src/Release-Process-Walkthrough.md:130]: Shouldn't this be `19.09`?

Also, isn't `nixpkgs-xx.xx-darwin` missing from the list?

Comment (jonringer) [reply src/Release-Process-Walkthrough.md:130]: yes, yes it is

File: src/Release-Process-Walkthrough.md [hunk 2]
```diff
@@ -143,12 +143,13 @@ git push upstream 20.03-pre
 
 - Release Nix (currently only Eelco Dolstra can do that). [Make sure fallback is updated.](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/installer/tools/nix-fallback-paths.nix)
 
-- [Update README.md with new stable NixOS version information.](https://github.com/NixOS/nixpkgs/commit/40fd9ae3ac8048758abdcfc7d28a78b5f22fe97e)
+- [Update README.md with new stable NixOS version information.](https://github.com/NixOS/nixpkgs/commit/40fd9ae3ac8048758abdcfc7d28a78b5f22fe97e) (for both `master` and `release-19.09` branches)
```
Comment (jonringer) [inline src/Release-Process-Walkthrough.md:146]:
```suggestion
- [Update README.md with new stable NixOS version information.](https://github.com/NixOS/nixpkgs/commit/40fd9ae3ac8048758abdcfc7d28a78b5f22fe97e) (for both `master` then backport to `release-19.09` branch)
```

File: src/Release-Process-Walkthrough.md [hunk 3]
```diff
@@ -143,12 +143,13 @@ git push upstream 20.03-pre
...
 
-- Change `stableBranch` to `true` in Hydra and wait for the channel to
+- Two days before expected release date, change `stableBranch` to `true` in Hydra and wait for the channel to
```
Comment (jonringer) [inline src/Release-Process-Walkthrough.md:148]: The "Two days before expected release date" should probably moved up to the header. All of the items should be "primed" before attempting a release.

File: src/Release-Process-Walkthrough.md [hunk 4]
```diff
@@ -186,7 +187,7 @@ You should include the following information:
 - Number of commits for the release:
 
 ```sh
-git log release-19.03..release-19.09 --format=%an | wc -l
+git log origin/release-19.03..release-19.09 --format=%an | wc -l
```
Comment (jonringer) [inline src/Release-Process-Walkthrough.md:190]: I think `upstream` was used in the other examples
```suggestion
git log upstream/release-19.03..upstream/release-19.09 --format=%an | wc -l
```

File: src/Release-Process-Walkthrough.md (hunk 1 above)
Comment (nrdxp) [reply src/Release-Process-Walkthrough.md:130]: is darwin relevant to a Linux Distro release though?

Comment (cole-h) [reply src/Release-Process-Walkthrough.md:130]: No, but it's relevant to the release all the same.

**Review by jonringer — APPROVED** [review]

**Pull request merged**